### ISSUE TITLE
No report of missing translation when using `marker("key")`

### ISSUE THREAD

**Issue by SimonSimCity** [svoboda-rabstvo/ngx-translate-lint#97]
# Bug report

## Describe the bug

The command does not report an error if a translation key is required using the following typescript:

```ts
import { marker } from "@biesbjerg/ngx-translate-extract-marker";

marker("some-random-translation-key");
```

## To Reproduce

- Add the package `@biesbjerg/ngx-translate-extract-marker`
- Define a key in your translation files which you do not use.
- Define a marker for it.
- Running the linter you should see no warning anymore.
- Define a marker for a translation key which is not translated.
- Running the linter you do not see an error, as you would if you define a non-translated key in your HTML template.

## Expected behavior

The command should fail when defining a marker for a translation key that does not exist.

## Screenshots

Not applicable.

## Additional context

I can verify that those keys are used in some way, because I can use them to suppress a notification of `Key: 'some-other-random-translation-key' doesn't exist in project'`

**Comment by SimonSimCity** [comment]
Maybe good for understanding the full story:

We use the package https://www.npmjs.com/package/@biesbjerg/ngx-translate-extract to automatically extract the language definitions needed. Since the translation files are auto-generated now, and the package does not inspect dynamic values passed into the `TranslateService`, they advocate the use of https://www.npmjs.com/package/@biesbjerg/ngx-translate-extract-marker where you can manually define what values are possible.

**Comment by svoboda-rabstvo** [comment]
Hi @SimonSimCity, I checked your app. I can create sepparete rule on config file for your case. Can't include this flow by default becuase `marker` can name differents function, not only from `@biesbjerg/ngx-translate-extract-marker`. What do you think about this?

**Comment by SimonSimCity** [comment]
A separate rule would already fix it for me, and since it's not the default way to do it, this would be OK (also kind of expected) for me.

Having to use the exact name of the function makes it even more interesting when you use an alternative name when importing it, right? 😅  Since this is a scenario announced on the main readme file, I guess many people are using it: https://github.com/biesbjerg/ngx-translate-extract#marker-function

```ts
import { marker as _ } from "@biesbjerg/ngx-translate-extract-marker";

_("some-random-translation-key");
``` 

Another option would be to build the project and include this in some kind of e2e testing, where all translation keys loaded are collected and in the end are checked against the keys available. Much like a code-coverage check ...

Again another option would be to take a closer look into how the package `@biesbjerg/ngx-translate-extract` actually finds those calls: https://github.com/biesbjerg/ngx-translate-extract/blob/master/src/parsers/marker.parser.ts

This last might be the best option, since you'll then find the same keys as the `ngx-translate-extract` command does.

**Comment by gastdozent91** [comment]
We have the same problem in our project, we try to mark our enums with comments like 
`/** marker(translation.key) */` and 
dynamic constructed keys like @SimonSimCity mentions above.

command:
`ngx-translate-lint -c ./ngx-translate-lint.config.json`

config:
```
{
  "rules": {
    "keysOnViews": "error",
    "zombieKeys": "warning",
    "misprint": "warning",
    "maxWarning": "0",
    "misprintCoefficient": "0.9",
    "ignoredKeys": [],
    "ignoredMisprintKeys": []
  },
  "project": "./src/app/**/*.{html,ts}",
  "languages": "./scripts/*.json"
}
```

What surprises me though is, that zombie keys (missing translation keys in project) are found, but not missing translations.

**Comment by svoboda-rabstvo** [comment]
Hi @SimonSimCity and @gastdozent91 

Mb's solution for that case will be allowing to add custom regexp to find translate keys ?

I think it's good idea
